psalmist_frontend/src/firebase/utils.js

```diff
@@ -0,0 +1,8 @@
+export const firebaseConfig = {
+    apiKey: "AIzaSyC2D2B3mFBRt_jd4OJQ2UwRDDvKDl-qThU",
```

Comment:
It is a good practice to have all of these values in your `.env` files and not committed